modules/admin_manual/pages/configuration/server/security/oauth2.adoc

```diff
@@ -96,6 +96,19 @@ To install the application, place the content of the OAuth2 app inside your inst
 
 To enable token-only based app or client logins in `config/config.php` set `token_auth_enforced` to `true`.
 
+==== Trusting Clients
+
+Since version 0.5.0 of the OAuth2 app, you can mark clients as trusted. This will have the effect that the consent step in the authentication process will be skipped for this client.
+
+image:configuration/server/security/oauth2_configuration.png[image]
+
+If you want to mark an existing client as trusted, you have to copy the `Client Identifier (ID)` and the `Client Secret`, then delete the existing entry and finally add it again via the occ comand. Follow this link regarding xref:configuration/user/oidc/oidc.adoc[ownCloud client config details].
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} oauth2:add-client <name> <client-id> <client-secret> <redirect-url> [<allow-sub-domains> [<trusted>]
+----
```

Comment:
I fixed how an occ command is added in docs **BUT**:

We do not have any entry in our occ documentation about an oauth2 command set ?!